yarn-project/circuits.js/src/structs/private_circuit_public_inputs.ts

```diff
@@ -56,6 +50,26 @@ export class PrivateCircuitPublicInputs {
      * New L2 to L1 messages created by the corresponding function call.
      */
     public newL2ToL1Msgs: Fr[],
+    /**
+     * Hash of the encrypted logs emitted in the current kernel iteration.
+     * Note: Represented as an array of 2 fields in order to fit in all of the 256 bits of sha256 hash.
+     */
+    public encryptedLogsHash: Fr[],
```

Comment:
Note that you can type this as `[Fr, Fr]` so ts knows it's exactly 2 fields. Depends on what's easier when you use this.

Reply:
Good point. Addressed in [5822751](https://github.com/AztecProtocol/aztec-packages/pull/636/commits/582275182ee87caa2328443b3f4de568616be378)